modules/claims_api/app/sidekiq/claims_api/claim_uploader.rb

```diff
@@ -17,14 +17,14 @@ def perform(uuid)
       doc_type = claim_object.is_a?(ClaimsApi::SupportingDocument) ? 'L023' : 'L122'
 
       if auto_claim.evss_id.nil?
-        ClaimsApi::Logger.log('claims_uploader', detail: "evss id: #{auto_claim&.evss_id} was nil, for uuid: #{uuid}")
+        ClaimsApi::Logger.log('claim_uploader', detail: "evss id: #{auto_claim&.evss_id} was nil, for uuid: #{uuid}")
```

Comment:
this is interesting. what's the intent with moving to `claim_uploader`? assuming it's all for consistency. 

also wonder if `claim_uploader` is vague. Perusing the logs for the last 4 hours I see none so I guess it's fine but wondering if we want to distinguish it a bit more so we know it's our team / code? Ultimately when we go to LHDI this would probably be fine though, maybe it's an unwarranted concern. Glad to leave it.